tests/appsec/test_blocking_addresses.py

```diff
@@ -134,6 +135,21 @@ def test_response_cookies(self):
         assert False
 
 
+def validate_custom_event_tag(expected_value="", present=True):
+    def wrapper(span):
+        tag = "appsec.events.system_tests_appsec_event.value"
+        if present:
+            assert tag in span["meta"], f"Can't find {tag} in span's meta"
+            value = span["meta"][tag]
+            if value != expected_value:
```

Comment:
It's unclear to me what the difference between raising an exception and asserting is in this context, but I was wondering if replacing the `if/raise` with `assert value == expected_value`  would have the same effect?

Reply:
I replaced that by an assert as suggested.